src/main/java/no/nav/veilarbperson/utils/PersonV2DataMapper.java

```diff
@@ -55,37 +56,37 @@ public static PersonNavnV2 navnMapper(List<HentPerson.Navn> personNavn) {
                 .setForkortetNavn(ofNullable(navn).map(HentPerson.Navn::getForkortetNavn).orElse(null));
     }
 
-    public static Familiemedlem familiemedlemMapper(HentPerson.Familiemedlem familiemedlem, boolean isEgenAnsatt, Bostedsadresse personsBostedsadresse, AuthService authService) {
+    public static Familiemedlem familiemedlemMapper(HentPerson.Familiemedlem familiemedlem, boolean erEgenAnsatt, Bostedsadresse personsBostedsadresse, AuthService authService) {
         HentPerson.Navn navn = getFirstElement(familiemedlem.getNavn());
         Fnr medlemFnr = ofNullable(getFirstElement(familiemedlem.getFolkeregisteridentifikator()))
                 .map(HentPerson.Folkeregisteridentifikator::getIdentifikasjonsnummer).map(Fnr::of).orElse(null);
-        String gradering = ofNullable(getFirstElement(familiemedlem.getAdressebeskyttelse())).map(HentPerson.Adressebeskyttelse::getGradering).orElse(null);
+        LocalDate fodselsdato = ofNullable(getFirstElement(familiemedlem.getFoedsel())).map(HentPerson.Foedsel::getFoedselsdato).orElse(null);
+        AdressebeskyttelseGradering gradering = ofNullable(getFirstElement(familiemedlem.getAdressebeskyttelse()))
```

Comment:
Bør kanskje ta vare på alle `AdressebeskyttelseGradering` i stede for `getFirstElement`, noe sånt:

```
List<AdressebeskyttelseGradering> graderinger = familiemedlem.getAdressebeskyttelse().stream()
                .map(HentPerson.Adressebeskyttelse::getGradering)
                .map(AdressebeskyttelseGradering::mapGradering).collect(toList());

boolean harAdressebeskyttelse = graderinger.stream()
        .anyMatch(gradering ->
                AdressebeskyttelseGradering.FORTROLIG.equals(gradering)
                        || AdressebeskyttelseGradering.STRENGT_FORTROLIG.equals(gradering)
                        || AdressebeskyttelseGradering.STRENGT_FORTROLIG_UTLAND.equals(gradering)
        );
```

Reply:
Her adressebeskyttelse liste er fra pdl og den inneholder ikke kun liste over graderinger men liste over historiske adressebeskyttelser/graderinger og historiske metadata. Vi henter kun først element fra liste fordi vi har ikke hatt behov for historiske data til bruker da vi sender false til historikk flag til pdl grapgql, så får vi kun data som er gyldig til bruker i responsen. men reponsen til informasjon element blir sendt i liste selv om liste inneholder kun den ene gyldige verdi/object.